src/phoenix/db/models.py

```diff
@@ -230,6 +248,30 @@ def _(element: Any, compiler: Any, **kw: Any) -> Any:
     )
 
 
+class TextContains(expression.FunctionElement[str]):
+    inherit_cache = True
+    type = String()
+    name = "text_contains"
+
+
+@compiles(TextContains)  # type: ignore
+def _(element: Any, compiler: Any, **kw: Any) -> Any:
+    string, substring = list(element.clauses)
+    return compiler.process(string.contains(substring), **kw)
+
+
+@compiles(TextContains, "postgresql")  # type: ignore
+def _(element: Any, compiler: Any, **kw: Any) -> Any:
+    string, substring = list(element.clauses)
+    return compiler.process(func.strpos(string, substring) > 0, **kw)
+
+
+@compiles(TextContains, "sqlite")  # type: ignore
+def _(element: Any, compiler: Any, **kw: Any) -> Any:
+    string, substring = list(element.clauses)
+    return compiler.process(func.text_contains(string, substring) > 0, **kw)
```

Comment:
Same as above.

Reply:
I'll just be linking it to the [docs](https://docs.sqlalchemy.org/en/20/core/compiler.html). I'm not sure how to explain this without confusing ppl.